### DISCUSSION TITLE
How to change the statusline style?

### DISCUSSION THREAD

**Discussion by niclashoyer** [NvChad/NvChad#1410] [Q&A] [answered]
After the latest updates I can't change the statusline style anymore. The default config does not contain these settings now, does it?

How is one supposed to change this now?

I tried

```
M.ui = {
  statusline = {
    separator_style = "arrow",
  },
}
```

**Comment by siduck** [comment]
read the docs!

https://nvchad.github.io/config/nvchad_ui

**Reply by niclashoyer** [reply]
Ah I'm so sorry. NvChad is moving so blazingly fast. I just hadn't discovered these docs yet 😶‍🌫️

**Reply by ElBe-Plaq** [reply]
Error 404

**Reply by ElBe-Plaq** [reply]
Here's the new one: https://nvchad.com/docs/config/nvchad_ui

**Reply by GhostOps77** [reply, marked as answer]
that site now just says
> **Read** `:h nvui`
